src/test/kotlin/com/esosa/f5pi_backend/controllers/implementations/UserControllerTest.kt

```diff
@@ -1,61 +1,80 @@
 package com.esosa.f5pi_backend.controllers.implementations
 
 import com.esosa.f5pi_backend.controllers.base.BaseIntegrationTest
-import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
 import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get
 import org.springframework.http.MediaType
 import org.springframework.test.context.jdbc.Sql
-import org.springframework.test.web.servlet.result.MockMvcResultHandlers
 import org.springframework.test.web.servlet.result.MockMvcResultMatchers.*
 import java.util.*
+import java.util.stream.Stream
 
-class UserControllerTest : BaseIntegrationTest()  {
- @Test
- fun `should return bad request when user does not exists`() {
-  val userId = UUID.randomUUID()
-    mockMvc.perform(
-     get("/api/v1/users/{userId}/fields", userId.toString())
-        .param("pageNumber", "0")
-        .param("pageSize", "2")
-        .param("sortAttribute", "createdAt")
-        .param("sortOrder", "desc")
-    )
-     .andExpect(status().isBadRequest)
-     .andExpect(content().contentType(MediaType.APPLICATION_JSON))
-     .andExpect(jsonPath("$.message").value("User with id $userId does not exist"))
- }
-
+class UserControllerTest : BaseIntegrationTest() {
+    companion object UserTestData {
+        val USER_UUID_WITH = UUID.fromString("58fae25b-ea38-4e7b-ab2d-9f555a67836b")
+        val USER_UUID_WITHOUT = UUID.fromString("68fae25b-ea38-4e7b-ab2d-9f555a67836e")
```

Comment:
Queda medio confuso jajaja. Podrían ser así

```suggestion
        val USER_UUID_WITH_DATA = UUID.fromString("58fae25b-ea38-4e7b-ab2d-9f555a67836b")
        val USER_UUID_WITHOUT_DATA = UUID.fromString("68fae25b-ea38-4e7b-ab2d-9f555a67836e")
```
Igual es una boludez